documentation/source/quickstart.rst

```diff
@@ -18,104 +18,92 @@ Cocotb has the following requirements:
+.. code-block:: bash
 
-Native Linux Installation
--------------------------
+    sudo yum install make gcc gcc-c++ libstdc++-devel swig python3 python3-devel python3-pip
 
-The following instructions will allow building of the cocotb libraries
-for use with a 64-bit native simulator.
+macOS Packages
+^^^^^^^^^^^^^^
 
-If a 32-bit simulator is being used then additional steps are needed, please see
-`our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
+You need a few packages installed to get cocotb running on macOS.
+Installing a package manager really helps things out here.
 
-Debian/Ubuntu-based
-^^^^^^^^^^^^^^^^^^^
+`Brew <https://brew.sh/>`_ seems to be the most popular, so we'll assume you have that installed.
 
 .. code-block:: bash
 
-    sudo apt-get install git make gcc g++ swig python-dev
+    brew install python icarus-verilog gtkwave
 
-Red Hat-based
-^^^^^^^^^^^^^
+.. seealso::
 
-.. code-block:: bash
+   For more installation options (also for 32-bit simulators) please see `our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
 
-    sudo yum install gcc gcc-c++ libstdc++-devel swig python-devel
 
+Simulator Installation
+^^^^^^^^^^^^^^^^^^^^^^
 
-Windows Installation
---------------------
+For detailed instructions and the list of supported simulators see :ref:`simulator-support`.
 
-Download the MinGW installer from https://osdn.net/projects/mingw/releases/.
-
-Run the GUI installer and specify a directory you would like the environment
-installed in. The installer will retrieve a list of possible packages, when this
-is done press "Continue". The MinGW Installation Manager is then launched.
+.. note::
 
-The following packages need selecting by checking the tick box and selecting
-"Mark for installation"
+    Ensure that path to the simulator executable is appended to the ``PATH`` environment variable.
 
-.. code-block:: bash
 
-    Basic Installation
-      -- mingw-developer-tools
-      -- mingw32-base
-      -- mingw32-gcc-g++
-      -- msys-base
+.. _installation_via_pip::
 
-From the Installation menu then select "Apply Changes", in the next dialog
-select "Apply".
+Cocotb Installation via PIP
+---------------------------
 
-When installed a shell can be opened using the :file:`msys.bat` file located under
-the :file:`<install_dir>/msys/1.0/`
+.. versionadded:: 1.2
 
-Python can be downloaded from https://www.python.org/downloads/windows/.
-Run the installer and download to your chosen location.
+Cocotb can be installed by running
 
-It is beneficial to add the path to Python to the Windows system ``PATH`` variable
-so it can be used easily from inside Msys.
+.. code-block:: bash
 
-Once inside the Msys shell commands as given here will work as expected.
+    pip install cocotb  # use `pip3` in case multiple of Python/pip version installed
```

Comment:
```suggestion
    pip install cocotb  # use `pip3` in case multiple versions of Python or pip are installed
```

Reply:
Maybe even better would be ``use `pip3` if available`` ?

Reply:
Or better opposite 
```bash
pip3 install cocotb  # use `pip` if `pip3` not available
```

Reply:
How about we just use
```
pip install cocotb
```
but add a note before the first time we mention `pip` like
```
.. warning::
    ``pip`` may belong to a different python installation to what you expect.
    Use ``pip -V`` to check.
    If this shows Python 2.7, use ``pip3`` or ``python3 -m pip`` in place of ``pip`` in the commands below.
```

Reply:
Sound OK. Yo sure **before** or rather close to `cocotb-config` note?

Reply:
You're right, your suggested placement makes more sense.

Reply:
:+1: If we see issues we move up ;-)